projects/plugins/social/src/js/store/selectors/jetpack-settings.js

```diff
@@ -3,6 +3,7 @@ const jetpackSettingSelectors = {
 	isModuleEnabled: state => state.jetpackSettings.publicize_active,
 	isUpdatingJetpackSettings: state => state.jetpackSettings.is_updating,
 	hasPaidPlan: state => ! state.showNudge,
+	getSiteRawUrl: state => state.siteRawUrl,
```

Comment:
If we're just getting this is the editor then I think we can use getSiteFragment and have it come from the editor's initial state, similarly to how [we do with the publicizeConnectionsUrl](https://github.com/Automattic/jetpack/blob/2447e267e40f2d133a5c15c2320b776e4c395ec8/projects/js-packages/publicize-components/src/components/settings-button/index.js#L26)